config/config.template.toml

```diff
@@ -1,9 +1,9 @@
 [irc]
-nickname = "Mechasqueak3-tests[BOT]"
+nickname = "Mechasqueak3[BOT|SPARK]"
```

Comment:
Please revert this specific change. I would perfer the default for this configuration object NOT be the production name.